src/components/Books.js

```diff
@@ -2,32 +2,26 @@ import React, { useEffect, useState } from 'react';
 import BookItem from './BookItem';
 import { useSelector } from 'react-redux';
 import { useDispatch } from 'react-redux';
-import { addBook } from '../redux/books/books';
+import { saveBook, fetchBooks } from '../redux/books/books';
 
 const Books = () => {
-
-    // const [books, setBooks] = useState([]);
     const { books } = useSelector((state) => state.books);
 
     const dispatch = useDispatch();
     const [book, setBook] = useState({ title: '', author: '' });
 
     const handleSubmit = (e) => {
         e.preventDefault();
-        dispatch(addBook({ id: Date.now(), ...book }));
+        console.log(book);
+        dispatch(saveBook({ id: Date.now(), ...book }));
 
         // clear input fields after dispatch
         setBook({ title: '', author: '' });
     };
```

Comment:
- [OPTIONAL] In order to make sure that the form inputs fields are cleared after form submission, please make sure to add the `value` attribute to each input and pass the state as a value
  **_Suggestion:_**
   `<input
              type='text'
              placeholder='Book title'
              name='title'
              onChange={(e) => setBook({ ...book, title: e.target.value })}
              value={book.title} 
            />
            <input
              type='text'
              placeholder='Author'
              name='author'
              onChange={(e) => setBook({ ...book, author: e.target.value })}
              value={book.author}
            />`

 - [OPTIONAL] Kindly make sure to add the validation so the user can not be able to add the empty book (without title and author).

Reply:
thanks you for your comment.
I'll add those changes to improve my project.

Best regards